gobblin-compaction/src/main/java/gobblin/compaction/hivebasedconstructs/MRCompactionEntity.java

```diff
@@ -0,0 +1,28 @@
+package gobblin.compaction.hivebasedconstructs;
+
+import java.util.Properties;
+import lombok.Getter;
+
+
+/**
+ * Entity that stores information required for launching an {@link gobblin.compaction.mapreduce.MRCompactor} job
+ *
+ * {@link #primaryKeyInfo}: Comma delimited list of fields to use as primary key
+ * {@link #deltaInfo}: Comma delimited list of fields to use as deltaInfo
+ * {@link #location}: Location of files associated with table
+ * {@link #props}: Other properties to be passed to {@link gobblin.compaction.mapreduce.MRCompactor}
+ */
+@Getter
+public class MRCompactionEntity {
+  private final String primaryKeyInfo;
```

Comment:
I suggest that you use make the object stronger typed. If primaryKeyInfo and deltaInfo are lists of field names, then declare them as a List<String>, if location is a Path, then declare it as such. The fact that the current implementation of MRCompactor takes a comma-separated list is an implementation detail. In the future, if we move it to using typesafe config, then we can pass directly lists.